src/libs/actions/Report.js

```diff
@@ -514,9 +514,13 @@ function updateReportActionMessage(reportID, sequenceNumber, message) {
  *
  * @param {Number} reportID
  * @param {Object} reportAction
- * @param {String} notificationPreference On what cadence the user would like to be notified
+ * @param {String} [notificationPreference] On what cadence the user would like to be notified
  */
-function updateReportWithNewAction(reportID, reportAction, notificationPreference) {
+function updateReportWithNewAction(
+    reportID,
+    reportAction,
+    notificationPreference = CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS,
```

Comment:
Why are we setting the default to always?

Reply:
If you look in this file, there are some usages of this function that did not provide third parameter, so it made sense to provide a default to keep the code clean and annotate that this is indeed an optional parameter. I chose to make the default preference `ALWAYS` because that's what it would be unless the user specified otherwise.

Reply:
Got it